### PR TITLE
Add Django 2.0; drop older Djangos that are or soon will be out of support

### PR DESCRIPTION
**Pull request by dpoirier**

### CODE REVIEW

File: runtests.py [hunk 1]
```diff
@@ -52,7 +52,7 @@
         ],
         # Load models directly to pick up test-only models
         # See: http://stackoverflow.com/a/25267435/347942
-        MIGRATION_MODULES={'treenav': 'treenav.migrations_not_run_during_tests'},
+        MIGRATION_MODULES={'treenav': None},
```
Comment (ironfroggy) [inline runtests.py]: Does the comment still apply?

File: tox.ini [hunk 2]
```diff
@@ -1,34 +1,39 @@
 [tox]
-envlist = {py27,py34,py35}-{dj1.8,dj1.9,dj1.10},py33-dj1.8,flake8,coverage,docs
+envlist = {py27,py34,py36}-dj1.11,{py34,py36}-dj2.0,{py27,py36}-flake8,coverage,docs
```
Comment (ironfroggy) [inline tox.ini]: 3.4 but not 3.5?

File: runtests.py (hunk 1 above)
Comment (dpoirier) [reply runtests.py]: Yes, the hack just needs updating for newer Django per the same stackoverflow discussion.

File: tox.ini (hunk 2 above)
Comment (dpoirier) [reply tox.ini]: I figured py34 and py36 would cover it, but we could add py35 too if you think it's necessary.

**Pull request merged**